test/functionalTest/cases/2135_disable_custom_notifications_cli/disble_custom_notifications_cli.test

```diff
@@ -0,0 +1,363 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Disable custom notifications CLI
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0 IPv4 -disableCustomNotifications
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# This is a a variant of case 2015 notification_templates_many_notifications.test using
+# the disableCustomNotifications CLI
+#
+# 01. Create E1/T1
+# 02. Create E1/T2
+# 03. Create E1/T3
+# 04. Create regular subscription for E1 -> 3 entities in one notification
+# 05. Create same subscription but with 'httpExtended' instead of 'http' -> 3 entities in one notification
+# 06. Create extended subscription for E1 -> 3 entities in one notification
```

Comment:
I miss some kind of clarification here, about why the broker behaves the way it behaves.
Something like:

    -> 3 entities in one notification (as the broker treats httpExtended as http when started with -disableCustomNotifications)